client/components/Home/index.jsx

```diff
@@ -1,65 +1,147 @@
-                    </li>
-                    <li>
-                        <a href="https://w3c.github.io/aria-at/results/">
-                            Review test results
-                        </a>
-                    </li>
-                    <li>
-                        <a href="https://github.com/w3c/aria-at/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22">
-                            Fix a good first issue
-                        </a>
-                    </li>
-                </ul>
+                        <h2>Get involved</h2>
+                        <ul>
+                            <li>
+                                <a href="https://www.w3.org/community/aria-at/">
+                                    Join the community group
+                                </a>
+                                {' (see '}
+                                <a href="https://www.w3.org/community/about/faq/#how-do-i-join-a-group">
+                                    how to join a W3C community group
+                                </a>
+                                )
+                            </li>
+                            <li>
+                                <a href="https://github.com/w3c/aria-at/issues/162">
+                                    Sign up as a tester
+                                </a>
+                                {' for the Pilot Test (May 27 - June 2)'}
+                            </li>
+                            <li>
+                                <a href="https://github.com/w3c/aria-at/wiki/How-to-contribute-tests">
+                                    Write more tests
+                                </a>
+                            </li>
+                            <li>
+                                <a href="https://w3c.github.io/aria-at/review-test-plans/">
+                                    Review the assertions of existing test plans
+                                </a>
+                            </li>
+                            <li>
+                                <a href="https://w3c.github.io/aria-at/results/">
+                                    Review test results
+                                </a>
+                            </li>
+                            <li>
+                                <a href="https://github.com/w3c/aria-at/issues?q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22">
+                                    Fix a good first issue
+                                </a>
+                            </li>
+                        </ul>
+                    </section>
+                    <section className="app-screenshots">
+                        <div className="container">
+                            <div className="description">
+                                <h2>Test Management</h2>
+                                <p>
+                                    This app allows users to create Test Cycles
```

Comment:
Maybe we should write to the reader with the assumption that they are the user. So, say "You can create Test Cycles" or similar